packages/graphql-tag-pluck/src/visitor.ts

```diff
@@ -194,6 +195,15 @@ export default (code: string, out: any, options: GraphQLTagPluckOptions = {}) =>
     );
   };
 
+  const addTemplateLiteralToResult = (content: PluckedContent) => {
+    const cacheKey = `end/${content.end}/start/${content.start}/${content.content}`;
```

Comment:
I thought maybe we could hash this, but I actually see no benefit in it and currently, the code does not rely on anything Node.js related

Reply:
Looks good!